### PR TITLE
docs: edits to vignettes

### PR DESCRIPTION
**Pull request by IndrajeetPatil**
Marking this as a draft for now, since there might be some other edits that I can include in this PR.

### CODE REVIEW

**Comment by IndrajeetPatil** [comment]
@lorenzwalthert This should be good to go.

File: vignettes/third-party-integrations.Rmd [hunk 1]
```diff
@@ -34,7 +34,7 @@ styler functionality is available in other tools, most notably
 
 -   in the *format-all* command for Emacs in [emacs-format-all-the-code](https://github.com/lassik/emacs-format-all-the-code).
 
--   As a [Jupyterlab code formatter](https://jupyterlab-code-formatter.readthedocs.io/en/latest/installation.html#r-code-formatters).
+<!-- FIXME -   As a [Jupyterlab code formatter](https://jupyterlab-code-formatter.readthedocs.io/en/latest/installation.html#r-code-formatters). -->
```
Comment (IndrajeetPatil) [inline vignettes/third-party-integrations.Rmd:37]: I can't find the correct link for this, if one even exists.

If not, I can just remove it.

Comment (lorenzwalthert) [reply vignettes/third-party-integrations.Rmd:37]: https://ryantam626.github.io/jupyterlab_code_formatter/index.html#

**Comment by lorenzwalthert** [comment]
Thanks for starting this 😎

File: R/ui-caching.R [hunk 2]
```diff
@@ -61,7 +61,7 @@ NULL
 #'
 #' Gives information about the cache. Note that the size consumed by the cache
 #' will always be displayed as zero because all the cache does is creating an
-#' empty file of size 0 bytes for every cached expression. The innode is
+#' empty file of size 0 bytes for every cached expression. The inode is
```
Comment (IndrajeetPatil) [inline R/ui-caching.R:64]: https://en.wikipedia.org/wiki/Inode

File: R/ui-styling.R [hunk 3]
```diff
@@ -1,4 +1,4 @@
-#' @api
+#' @keywords api
```
Comment (IndrajeetPatil) [inline R/ui-styling.R:1]: _roxygen2_ doesn't have any such tag, so this leaves a warning.

File: vignettes/caching.Rmd [hunk 4]
```diff
@@ -18,11 +18,11 @@ knitr::opts_chunk$set(
 library(styler)
 ```
 
-This is a developer vignette to explain how caching works and what we learned on the way. To use the caching feature, please have a look at the README.
```
Comment (IndrajeetPatil) [inline vignettes/caching.Rmd:21]: README doesn't have anything about caching anymore.

**Comment by IndrajeetPatil** [comment]
@lorenzwalthert I think this is ready for a review.

**Review by lorenzwalthert — APPROVED** [review]
Great. Nothing to add from my side here.

**Comment by lorenzwalthert** [comment]
Thanks 😊

**Pull request merged**